MetadataExtractor.Tests/Formats/Mov/MovMetadataReaderTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Drew Noakes and contributors. All Rights Reserved. Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using MetadataExtractor.Formats.QuickTime;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace MetadataExtractor.Tests.Formats.Mov
+{
+    /// <summary>Unit tests for <see cref="QuickTimeMetadataReader"/>.</summary>
+    public sealed class MovMetadataReaderTest
+    {
+
+        [Fact]
+        public void ExtractMetadataUsingStream()
+        {
+            using var stream = TestDataUtil.OpenRead("Data/with-gps.mov");
```

Comment:
Thank you for writing tests. On other repos I find not enough PRs have tests! However in this project, because of the sheer number of data formats required, instead of adding media to the source repository we use a separate media library which tracks all output for all files within, across both the .NET and Java implementations.

Could you please remove the `.mov` file and unit tests from this PR? You'll need to edit your PR branch's history. Let me know if I can help with that.

Instead, let's add the file to this repo:

https://github.com/drewnoakes/metadata-extractor-images

Reply:
Looks like that file is already there: https://github.com/drewnoakes/metadata-extractor-images/commit/a5895dc0b5b3b47df740bffea5451b8e951d26ad

Reply:
Yeah, that's where I got it from. it was referenced in the issue thread.
Yup, I'll remove it from my branch's history.